poclu/poclu.h

```diff
@@ -369,6 +369,14 @@ int poclu_load_program_multidev (cl_platform_id platform,
  */
 POCLU_API cl_int POCLU_CALL poclu_show_program_build_log (cl_program program);
 
+/**
+ * \brief check if a device supports a particular version of SPIR-V
+ * @param il [in] a string like "SPIR-V_1.2" of the requested version
```

Comment:
We try to converge to the LLVM doxygen conventions. Pls use \param (or \p) and \return in new comments. No need for the \brief tag, but an empty comment line after it instead.